lib/solidus_subscriptions/engine.rb

```diff
@@ -11,9 +12,17 @@ class Engine < Rails::Engine
     end
 
     config.autoload_paths << config.root.join('app', 'jobs')
-    config.autoload_paths << config.root.join('lib')
 
-    initializer 'solidus_subscriptions.configs', before: "spree.register.payment_methods" do
+    initializer 'require subscription lib helpers' do
+      require 'solidus_subscriptions/permitted_attributes'
```

Comment:
Are you requiring this line twice? Why do we need an initializer block for requiring files?